iRecruit-express-app/memcache.js

```diff
@@ -0,0 +1,19 @@
+const cache = {};
+
+function get(key) {
+  return cache[key];
+  // retrieves data from the cache based on the given key
+}
+
+function set(key, value, expiration = null) {
```

Comment:
Still missing actual TTL implementation, if merging incomplete code, its standard practice to write a comment explaining so.

Reply:
Oh noted and taken care of!